firedrake/tsfc_interface.py

```diff
@@ -134,19 +129,16 @@ def __init__(self, form, name, parameters, number_map, interface, coffee=False,
         if self._initialized:
             return
         tree = tsfc_compile_form(form, prefix=name, parameters=parameters,
-                                 interface=interface, coffee=coffee,
+                                 interface=interface,
                                  diagonal=diagonal, log=PETSc.Log.isActive())
         kernels = []
         for kernel in tree:
-            # Set optimization options
-            opts = default_parameters["coffee"].copy()
```

Comment:
Check these parameters aren't coming from "legacy" optimisation parameters (named coffee)

Reply:
Tracing this back, all this does is set `optlevel` to `Ov`. We manage this perfectly well in PyOP2 and gcc also doesn't seem to know what that flag does anyway. I think that we can probably stop worrying about this.